core-java-modules/core-java-io-conversions-2/src/test/java/com/baeldung/filetofrombytearray/FileToByteArrayUnitTest.java

```diff
@@ -0,0 +1,63 @@
+package com.baeldung.filetofrombytearray;
+
+import static org.junit.Assert.assertArrayEquals;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+import org.apache.commons.io.FileUtils;
+import org.apache.commons.io.IOUtils;
+import org.junit.jupiter.api.Test;
+
+class FileToByteArrayUnitTest {
+
+    private static final String FILE_NAME = "src/test/resources/sample.txt";
```

Comment:
Question here: What happens if we use Windows OS and *nix system? Shouldn't we use File.Separator?

Reply:
Good question! For simplicity, I used the same file and syntax used in other unit tests.

Reply:
Done